pages/stonewall.jsx

```diff
@@ -171,39 +171,41 @@ const Stonewall = () => {
                 }
             `}
             >
-            <div id="stonewall-wrap">
-                <div id="title">
-                    {"STONEWALL".split('').map((letter, i) => {
-                        return <h2 key={i} style={{opacity: 0, animation: 'fadeInDown 1s ease !important', animationDelay: `${(0.15 * (i + 1))}s !important`, animationFillMode: 'forwards !important'}}>{letter}</h2>
-                    })}
-                <h3>From the Daily Bruin</h3>
+            <div id="stonewall-wrap" style={{ fontFamily: 'Source Sans Pro', fontSize: '10', fontWeight: 400, lineHeight: '20px', letterSpacing: '0em', textAlign: 'center' }}>
```

Comment:
Try not to add styles here, the way we're handling styles isn't the best but we should try to keep it consistent. We have a style sheet further up in the file. This style also effects the whole page because it wraps a lot.